content/workflows/fairness-hiding-protected-attribute.md

```diff
@@ -0,0 +1,12 @@
++++
+title= "Hiding Protected Attribute"
+images =  ["/workflows/images/fairness-hiding-protected-attribute.png"]
+type = "workflows"
+blog_link =  "/blog/2023/2023-08-31-fairness-hiding-protected-attribute/"
+video = ""
+download = "fairness-hiding-protected-attribute.ows"
+workflows = ["Fairness"]
+weight = 1005
++++
+
+Many would question why we would use fairness algorithms instead of simply removing the protected attribute to avoid bias towards certain groups in that attribute. This workflow illustrates why that is not a good idea. We will compare the predictions of a model using Reweighing to that of a model learning and predicting on data with the protected attribute removed. We will visualize and compare the results using a Box Plot.
```

Comment:
My rewrite: Why would we use fairness algorithms instead of simply removing the protected attribute? We compare the predictions of a model using Reweighing to those of a model on data with the protected attribute removed. We use Box Plot to compute common fairness metrics.